### DISCUSSION TITLE
ADGUARD For Windows won't launch

### DISCUSSION THREAD

**Discussion by Julius771** [community/community#61743] [Other Feature Feedback, Questions, & Ideas]
Hello,  

When I double click on Adguard for Windows on my desktop, the program doesn't launch.  What is going on?

Julius

**Comment by queenofcorgis** [comment]
Hi @Julius771 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Adguard. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Comment by Julius771** [comment]
Lili,

 

Thank you for responding.  When I went to the ADGUARD website to submit a support ticket, it sent me to GITHUB.  I will re-visit ADGUARD to see what’s going on.  Thanks again.

 

Julius771
